### PR TITLE
Migrate tests from ScalaTest to Munit.

### LINKED ISSUE #228: Migrate tests to MUnit.
See https://github.com/circe/circe/issues/1582

### PR DESCRIPTION
**Pull request by diesalbla**
Resolves #228

### CODE REVIEW

File: shared/src/test/scala/io/circe/jackson/CirceSuite.scala [hunk 1]
```diff
@@ -4,25 +4,13 @@ import cats.instances.AllInstances
...
-    with AllInstances
-    with AllSyntax
-    with ArbitraryInstances
-    with EqInstances {
-
-  override def convertToEqualizer[T](left: T): Equalizer[T] =
-    sys.error("Intentionally ambiguous implicit for Equalizer")
+trait CirceSuite extends ScalaCheckSuite with AllInstances with AllSyntax with ArbitraryInstances with EqInstances {
 
   implicit def prioritizedCatsSyntaxEither[A, B](eab: Either[A, B]): EitherOps[A, B] = new EitherOps(eab)
```
Comment (travisbrown) [inline shared/src/test/scala/io/circe/jackson/CirceSuite.scala:15]: I don't think we need this now, do we?

Comment (diesalbla) [reply shared/src/test/scala/io/circe/jackson/CirceSuite.scala:15]: No. Dropped.

Comment (travisbrown) [reply shared/src/test/scala/io/circe/jackson/CirceSuite.scala:15]: :+1: Incidentally I think we can get rid of `AllInstances` now, too, thanks to Cats 2.2.

**Comment by travisbrown** [comment]
Looks good to me, thanks again!

**Pull request merged**